Core/Core/Store/Store.swift

```diff
@@ -261,6 +271,18 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate, Ob
         requested = true
         pending = true
         notify()
+
+        guard !offlineService.isOfflineModeEnabled() else {
```

Comment:
An `if` would be more straightforward in this case.
```suggestion
        if offlineService.isOfflineModeEnabled() {
```

Reply:
I was thinking about this too. Will change it